plugins/scaffolder/src/legacy/TemplateEditorPage/TemplateEditorForm.tsx

```diff
@@ -195,7 +196,7 @@ export function TemplateEditorForm(props: TemplateEditorFormProps) {
             formData={data}
             onChange={e => onUpdate(e.formData)}
             onReset={() => onUpdate({})}
-            finishButtonLabel={onDryRun && 'Try It'}
+            finishButtonLabel={onDryRun && 'Try iIt'}
```

Comment:
you probably meant the following, right? 😬

```suggestion
            finishButtonLabel={onDryRun && 'Try it'}
```